scim-sdk-common/src/main/java/de/captaingoldfish/scim/sdk/common/utils/EncodingUtils.java

```diff
@@ -0,0 +1,32 @@
+package de.captaingoldfish.scim.sdk.common.utils;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+
+
+/**
+ * This class provides methods for encoding data into specific formats.
+ */
+public class EncodingUtils
+{
+
+  /**
+   * This method takes the input string and URL Encodes it using java.net.URLEncoder
+   *
+   * @param value The value to be URL encoded.
+   * @return A URL encoded version of the input value
+   */
+  public static String urlEncode(String value)
```

Comment:
could you move this into the class `UtilityMethods`?
I would prefer to have fewer than more classes of this kind :-)

Reply:
Sure, no problem, I will make the change.